perceval/utils/statevector.py

```diff
@@ -239,6 +239,22 @@ def m(self):
     def n_max(self):
         return self._n_max
 
+    @staticmethod
+    def tensor_product(svd1, svd2, prob_threshold: float = 0):
```

Comment:
This is a good method of simplification ! Are you sure about the default value ? We don't want 1e-6/1e-16 instead of zero ? I don't see any UT for this method, can you implement one real quick ?